examples/project/dialog_box/test_2_input_field.py

```diff
@@ -34,3 +34,32 @@ def test_num_input():
     response = int(run_dialog_box(dbx))
     set_message(f"Entered number {response}")
     assert response == TEST_NUM, "The entered number is not correct"
+
+
+@pytest.mark.case_name("Radiobutton")
+def test_radiobutton():
+    dbx = DialogBox(
+        dialog_text=f"Select one item (1) from several and click the Confirm button",
+        title_bar="Radiobutton example",
+        widget=DialogBoxWidget(
+            DialogBoxWidgetType.RADIOBUTTON,
+            {"options": ["one", "two", "three"]},
```

Comment:
I suggest add `RadiobuttonInfo` and `CheckboxInfo` dataclasses in `dialog_box.py`. 

```python
@dataclass
class RadiobuttonInfo:
    filed: list[str | int | float]
```

And use these class instead of `dict` in `DialogBoxWidget`.

Reply:
Ready